src/mscorlib/mscorlib.csproj

```diff
@@ -148,7 +148,6 @@
   <ItemGroup>
     <!-- We want the sources to show up nicely in VS-->
```

Comment:
Sure. I guess it seemed to me like the comment was there to call out why the Link stuff was being done (since otherwise it's pretty standard).

LGTM.